contrib/msoffice/src/com/sun/jna/platform/win32/COM/util/office/MSOfficeWordDemo.java

```diff
@@ -0,0 +1,106 @@
+/* Copyright (c) 2014 Dr David H. Akehurst (itemis), All Rights Reserved
+ *
+ * This library is free software; you can redistribute it and/or
+ * modify it under the terms of the GNU Lesser General Public
+ * License as published by the Free Software Foundation; either
+ * version 2.1 of the License, or (at your option) any later version.
+ *
+ * This library is distributed in the hope that it will be useful,
+ * but WITHOUT ANY WARRANTY; without even the implied warranty of
+ * MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the GNU
+ * Lesser General Public License for more details.
+ */
+package com.sun.jna.platform.win32.COM.util.office;
+
+import java.io.File;
+
+import com.sun.jna.platform.win32.COM.COMException;
+import com.sun.jna.platform.win32.COM.util.Factory;
+import com.sun.jna.platform.win32.COM.util.office.word.ComIApplication;
+import com.sun.jna.platform.win32.COM.util.office.word.ComWord_Application;
+import com.sun.jna.platform.win32.COM.util.office.word.WdOriginalFormat;
+import com.sun.jna.platform.win32.COM.util.office.word.WdSaveFormat;
+
+public class MSOfficeWordDemo {
+
+	/**
+	 * @param args
+	 */
+	public static void main(String[] args) {
+		new MSOfficeWordDemo();
+	}
+
+	private String currentWorkingDir = new File("").getAbsolutePath() + File.separator;
+
+	public MSOfficeWordDemo() {
+		this.testMSWord();
+	}
+
+	public void testMSWord() {
+		ComWord_Application msWordObject = null;
+		ComIApplication msWord = null;
+		Factory factory = null;
+		try {
+			factory = new Factory();
+			msWordObject = factory.createObject(ComWord_Application.class);
+			msWord = msWordObject.queryInterface(ComIApplication.class);
+
+			System.out.println("MSWord version: " + msWord.getVersion());
+
+			msWord.setVisible(true);
+			// msWord.newDocument();
+			msWord.getDocuments().Open(currentWorkingDir + "jnatest.doc");
+			msWord.getSelection().TypeText("Hello from JNA! \n\n");
+			// wait 10sec. before closing
+			Thread.sleep(1000);
+			// save in different formats
+			// pdf format is only supported in MSWord 2007 and above
+			msWord.getActiveDocument().SaveAs("C:\\TEMP\\jnatestSaveAs.doc", WdSaveFormat.wdFormatDocument);
```

Comment:
If you feel motivated, fetch the TMP directory (GetTemp...something) and combine it here. Not everybody has a `C:\\TEMP`.